src/rebar_prv_xref.erl

```diff
@@ -104,8 +104,7 @@ prepare(State) ->
 
     %% Get list of xref checks we want to run
     ConfXrefChecks = rebar_state:get(State, xref_checks,
-                                     [exports_not_used,
-                                      undefined_function_calls]),
+                                     [undefined_function_calls]),
```

Comment:
shouldn't change the defaults like that, may break compat for some users

Reply:
I think we should consider it for 3.14.